src/RPCClient.js

```diff
@@ -29,7 +31,11 @@ async function request(url, method, params, options = {}) {
   }
   const { data } = res;
   if (data.error) {
-    throw new Error(`DAPI RPC error: ${method}: ${data.error.message}`);
+    if (data.error.data) {
+      throw new RPCError(`DAPI RPC error: ${method}: ${data.error.message}`, `${data.error.data}`);
```

Comment:
Actually `data` is a plain object, so you don't need to convert it to a string.

Reply:
modified